python/semantic_kernel/processes/process_edge_builder.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import TYPE_CHECKING
+
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+from semantic_kernel.processes.process_function_target_builder import ProcessFunctionTargetBuilder
+from semantic_kernel.processes.process_step_edge_builder import ProcessStepEdgeBuilder
+from semantic_kernel.utils.experimental_decorator import experimental_class
+
+if TYPE_CHECKING:
+    from semantic_kernel.processes.process_builder import ProcessBuilder
+
+
+@experimental_class
+class ProcessEdgeBuilder(KernelBaseModel):
+    """A builder for a process edge."""
+
+    source: "ProcessBuilder"
+    target: ProcessFunctionTargetBuilder | None = None
+    event_id: str
+
+    def __init__(self, source: "ProcessBuilder", event_id: str):
+        """Initializes a new instance of ProcessEdgeBuilder."""
+        super().__init__(source=source, event_id=event_id)  # type: ignore
+
+    def send_event_to(self, target: ProcessFunctionTargetBuilder) -> "ProcessEdgeBuilder":
```

Comment:
could we make this method accept the args of ProcessFunctionTargetBuilder as well as the object itself? makes writing your process a lot simpler!

Reply:
basically change this to:
```suggestion
    def send_event_to(self, target: ProcessFunctionTargetBuilder | ProcessStepBuilder, **kwargs) -> "ProcessEdgeBuilder":
```
or optional arguments from ProcessFunctionTargetBuilder